### ISSUE TITLE
The error messages return wrong lines for `Learn2_new.py`

### ISSUE THREAD

**Issue by georgemilosh** [georgemilosh/Climate-Learning#54]
I was running
python hyperparameter_optimization.py n_trials=100
I got an error message:
```
File "/ClimateDynamics/MediumSpace/ClimateLearningFR/gmiloshe/CESM/models/hypop/t2-bn-dr-l2/Learn2_new.py", line 1441, in create_model
    raise ValueError(f'Invalid length for argument {arg}')
ValueError: Invalid length for argument ['relu', None]
```

The problem is the line that is reported does not correspond to where the error happens precisely in the code. I've noticed this before and it is quite annoying when debugging.

**Comment by AlessandroLovo** [comment]
Is this the full traceback?

**Comment by AlessandroLovo** [comment]
My interpretation is that you are providing two activation functions `['relu', None]` but your network has more or less than 2 layers. That's why it says invalid length.

**Comment by georgemilosh** [comment]
No, as usual they are quite long. But this is the last part of the error message. I understand why I get this error message, I am just saying that there is something wrong with tracebacks in general. Tney often report wrong line number

**Comment by AlessandroLovo** [comment]
To me the line is correct: if you go at 1441 in Learn2_new.py you'll actually find `raise ValueError(f'Invalid length for argument {arg}')`

**Comment by georgemilosh** [comment]
Ok. Then it must have been another case when I was confused...
